README.md

```diff
@@ -1,4 +1,14 @@
-POC of new oVirt User Portal.
+# User Portal for oVirt
+Fancy looking UI for standard (non-admin) oVirt users.
+
+This project is meant to be full replacement for existing oVirt Basic User Portal.
+Revised list of Extended User Portal features will be implemented to ideally replace it as well.
+
+Performance bottlenecks of previous Basic User Portal are addressed.
+
+This project is not intended to be full-feature oVirt admin UI, it's focus is on standard user with no or limited administration skills or privileges.
+
+For list of features please refer the [Milestones](https://github.com/oVirt/ovirt-web-ui/milestones) or [Issues](https://github.com/oVirt/ovirt-web-ui/issues).
 
```

Comment:
I would maybe add some sentence like:
## Fist Milestone
The first milestone should support a subset of the functionality of the current Basic User Portal, namely:
- list of VMs (not pools)
- basic operations (start/stop/...)
- take console
- present VM details
For details: <link to the milestone>

## Second milestone
Should include all features of the Basic User Portal and include selected functionality of the Power User Portal, namely:
- add/edit VM (basic properties only)